iguana.corecontroller/src/test/java/org/aksw/iguana/cc/query/source/impl/FileSeparatorQuerySourceTest.java

```diff
@@ -45,17 +45,17 @@ public void sizeTest() {
 
     @Test
     public void getQueryTest() throws IOException {
-        assertEquals("QUERY 1 {still query 1}", this.querySource.getQuery(0));
-        assertEquals("QUERY 2 {still query 2}", this.querySource.getQuery(1));
-        assertEquals("QUERY 3 {still query 3}", this.querySource.getQuery(2));
+        assertEquals("QUERY 1 {\r\nstill query 1\r\n}", this.querySource.getQuery(0));
```

Comment:
Please add this case: separator: "###$", string: "a####$b" Result should actually be "a#" and "b" or (0,2), (6,1) respectively